tests/README.md

```diff
@@ -14,7 +14,7 @@ The TodoMVC project has a great many implementations of exactly the same app usi
 
 ## Running the tests
 
-These tests use Selenium 2 (WebDriver), via the JavaScript API (WebdriverJS).  In order to run the tests you will need to install the dependencies. Run the following command from within the `tests` folder:
+These tests use Selenium 2 (WebDriver), via the JavaScript API (WebdriverJS).  In order to run the tests you will need to install the dependencies.  NPM to be version 2.0.0 or greater, so upgrade it first with `npm install -g npm` if `npm --version` outputs anything less than 2.0.0. Run the following command from within the `tests` folder:
```

Comment:
It can be, but the part that specifies the folder comes next.  Should we break it out into a separate paragraph so that it's clearer that:
 > NPM to be version 2.0.0 or greater, so upgrade it first with `npm install -g npm` if `npm --version` outputs anything less than 2.0.0.

and 

> Run the following command from within the `tests` folder:

are separate sentences?